README.rst

```diff
@@ -27,84 +27,36 @@ This should fetch and install the Portier broker. The binary is installed into
 ``~/.cargo/bin/`` by default. Running the `portier-broker` binary requires
 a short configuration file. An example is provided in ``config.toml.dist``.
 
-.. _installed: https://doc.rust-lang.org/book/getting-started.html
-
-
-Configuration
--------------
-
-Here's an example configuration file:
-
-.. code-block:: toml
-
-    listen_ip = "127.0.0.1"
-    listen_port = 3333
-    base_url = "https://portier.example.com"
-    token_validity = 600
-
-    [[keys]]
-    file = "private.pem"
+To run the broker, invoke it with the path to a configuration file:
 
-    [store]
-    redis_url = "redis://127.0.0.1/5"
-    expire_sessions = 900
-    expire_cache = 3600
-    max_response_size = 8096
-
-    [smtp]
-    address = "localhost:25"
-
-    [sender]
-    name = "Portier"
-    address = "portier@example.com"
-
-    [providers . "gmail.com"]
-    discovery = "https://accounts.google.com/.well-known/openid-configuration"
-    client_id = "1234567890-example-client-id.apps.googleusercontent.com"
-    secret = "<your-secret-goes-here>"
-    issuer = "accounts.google.com"
-
-**listen_ip** and **listen_port** contain the port on which the broker listens.
-
-**base_url** contains the web origin for this broker instance.
+.. code-block:: shell
 
-**token_validity** is a value in seconds, that determines how long outgoing
-authentication tokens are allowed to live. Defaults to 600s, or 10 minutes.
+   $ portier-broker config.toml
 
-**keys** is a list of RSA private keys, with a ``file`` path for each key.
-Multiple keys can be used to implement key rotation. By default, the last key
-in the list will be used for signing the outgoing JWTs.
+You will also need a Redis server and an outgoing SMTP server.
```

Comment:
I think there's too much going on in this commit, making it hard to carefully review. The move from config-in-README to docs-in-TOML should probably be one commit on its own.